Cargo.toml

```diff
@@ -101,24 +100,48 @@ lto = true
 [workspace.dependencies]
 bytes = { version = "1" }
 drain = { version = "0.1", default-features = false }
-h2 = { version = "0.3" }
-http = { version = "0.2" }
-http-body = { version = "0.4" }
-hyper = { version = "0.14.32", default-features = false }
-prost = { version = "0.12" }
-prost-build = { version = "0.12", default-features = false }
-prost-types = { version = "0.12" }
-tokio-rustls = { version = "0.26", default-features = false, features = [
-    "ring",
-    "logging",
-] }
-tonic = { version = "0.10", default-features = false }
-tonic-build = { version = "0.10", default-features = false }
-tower = { version = "0.4.13", default-features = false }
+h2 = { version = "0.4" }
+http = { version = "1" }
+http-body = { version = "1" }
+hyper = { version = "1", default-features = false }
+hyper-util = { version = "0.1", default-features = false }
+prost = { version = "0.13" }
+prost-build = { version = "0.13", default-features = false }
+prost-types = { version = "0.13" }
+tonic = { version = "0.12", default-features = false }
+tonic-build = { version = "0.12", default-features = false }
+tower = { version = "0.4", default-features = false }
 tower-service = { version = "0.3" }
 tower-test = { version = "0.4" }
 
+[workspace.dependencies.http-body-util]
+version = "0.1"
+default-features = false
+features = ["channel"]
+
+[workspace.dependencies.tokio-rustls]
+version = "0.26"
+default-features = false
+features = ["logging", "ring"]
+
+# Temporarily patch `drain`. When linkerd/drain-rs#36 is released, this can
+# be removed.
+[patch.crates-io.drain]
+git = "https://github.com/linkerd/drain-rs.git"
+rev = "7678cbbf822c922fefa5c8c62070f52bea385fc9"
+# https://github.com/linkerd/drain-rs/commit/7678cbbf822c922fefa5c8c62070f52bea385fc9
+
+# Temporarily patch `http-body` and its sibling `http-body-util`, using a
+# fork in the `linkerd` Github oranization. When hyperium/http-body#140 and
+# hyperium/http-body#140 have been released, this can be removed.
+[patch.crates-io.http-body]
+git = "https://github.com/linkerd/http-body.git"
+rev = "c86632ea5c869f28701b8de386e5c19b4388462b"
+[patch.crates-io.http-body-util]
+git = "https://github.com/linkerd/http-body.git"
+rev = "c86632ea5c869f28701b8de386e5c19b4388462b"
```

Comment:
see https://github.com/hyperium/http-body/pull/149, this will be released in v0.1.3

Reply:
this has now been released, these patches have been removed from this branch!